Core/Categories/Category.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace Core.Categories
+{
+    public abstract class Category
```

Comment:
Codacy found an issue: [Convert this 'abstract' class to a concrete class with a protected constructor.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=6003477)